proto/google/events/cloud/audit/v1/events.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "title": "google.events.cloud.audit.v1",
+    "goPackage": "auditv1",
```

Comment:
I don't think we should have language-specific  keys in our schema, right?

Reply:
It is true that `goPackage` is not a field listed in the JSON schema spec; unfortunately we need it to generate the Go package (in the same way as Protocol Buffers Generator requires the `go_package` option). Alternatively we could infer this value from the `title` field for languages like Go, but it will not be very elegant (have to use hard coded prefixes).

Reply:
No issues with this then :)